### DISCUSSION TITLE
`force`,`release`, `pullup` and `pulldown` in cocotb

### DISCUSSION THREAD

**Discussion by GiuseppeDiGuglielmo** [cocotb/cocotb#3594] [Q&A]
Does cocotb provides `force`, `release`, `pullup` and `pulldown` statements as SystemVerilog does?

For example:
```
force EN = 1;
// ...
release EN = 0;
```
or
```
pulldown(EN);
// ...
pullup(EN);
```

See also the SV LRM for [force/release](https://fpga.mit.edu/6205/_static/F23/documentation/1800-2017.pdf#page=243) and  [pullup/pulldown](https://fpga.mit.edu/6205/_static/F23/documentation/1800-2017.pdf#page=810).

**Comment by marlonjames** [comment]
https://docs.cocotb.org/en/stable/writing_testbenches.html#forcing-and-freezing-signals
